deepchecks/checks/performance/regression_error_distribution.py

```diff
@@ -0,0 +1,113 @@
+from sklearn.base import BaseEstimator
+
+from deepchecks import CheckResult, Dataset, SingleDatasetBaseCheck, ConditionResult
+from deepchecks.utils.metrics import ModelType, task_type_validation
+from deepchecks.utils.strings import format_number
+
+__all__ = ['RegressionErrorDistribution']
+
+
+class RegressionErrorDistribution(SingleDatasetBaseCheck):
+    """Check regresstion error distribution.
+
+    Args:
+        n_top_samples (int): amount of samples to show which are of Largest under / over estimation errors.
+        alternative (str): Defines the alternative hypothesis to calculate the p value
+            ‘two-sided’: the kurtosis of the distribution underlying the sample is different from that of
+                         the normal distribution
+            ‘less’: the kurtosis of the distribution underlying the sample is less than that of
+                    the normal distribution
+            ‘greater’: the kurtosis of the distribution underlying the sample is greater than that of
+                        the normal distribution
+            (taken from: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.kurtosistest.html)
+            (Only in python 3.7>=)
+    """
+
+    def __init__(self, n_top_samples: int = 3, alternative: str = 'two-sided'):
+        super().__init__()
+        self.n_top_samples = n_top_samples
+        self.alternative = alternative
+
+    def run(self, dataset: Dataset, model: BaseEstimator) -> CheckResult:
+        """Run check.
+
+        Arguments:
+            dataset (Dataset): A dataset object.
+            model (BaseEstimator): A scikit-learn-compatible fitted estimator instance
+
+        Returns:
+           CheckResult:
+                - value is the kurtosis value (Fisher’s definition (normal ==> 0.0)) and the p score of it.
+                - display is histogram of error distirbution and the largest prediction errors.
+
+        Raises:
+            DeepchecksValueError: If the object is not a Dataset instance with a label
+        """
+        return self._regression_error_distribution(dataset, model)
+
+    def _regression_error_distribution(self, dataset: Dataset, model: BaseEstimator):
+        check_name = self.__class__.__name__
+        Dataset.validate_dataset(dataset, check_name)
+        dataset.validate_label(check_name)
+        task_type_validation(model, dataset, [ModelType.REGRESSION], check_name)
+
+        y_test = dataset.label_col
+        y_pred = model.predict(dataset.features_columns)
+
+        diff = y_test - y_pred
+        kurtosis_value = kurtosis(diff)
+        # alternative doesn't work on pytho3.6
+        try:
+            kurtosis_pvalue = kurtosistest(diff, alternative = self.alternative).pvalue
+        except TypeError:
+            kurtosis_pvalue = kurtosistest(diff).pvalue
+
+        n_largest_diff = diff.nlargest(self.n_top_samples)
+        n_largest_diff.name= n_largest_diff.name + '_pred_diff'
```

Comment:
Also show prediction. If showing a diff column, it needs to have a nice name such as "Target Prediction Difference"

Reply:
ok